### ISSUE TITLE
Игры 

### ISSUE THREAD

**Issue by Davibiyuu** [Davibiyuu/zashitacoil#285]
Понимаю, что очень важно найти разнообразные рейтинги для того, чтобы выбрать хорошие игровые автоматы, что ещё вы посоветовали бы изучить вы?

**Comment by diorjio** [comment]
Да, действительно, лучшим вариантом будет изучить различные рейтинги, где собраны самые проверенные и надёжные автоматы. Например, я доверяю вот этому сайту, где собраны лучшие https://alchevskpravoslavniy.ru/kazino-s-popolneniem-s-telefona/igrovye-avtomaty-s-minimalnymi-depozitami/	игровые автоматы с минимальными депозитами . Очень удобно, что на сайте они собраны по разным критериям, можно сравнить, есть удобная таблица, описана отдача RTP, минимальные депозиты, бонусные раунды, скаттеры, линии, разнообразные плюсы и минусы. Можно узнать, какая лицензия, есть таблица с бонусами. Поверьте, изучив данный сайт, вы на 100% будете уверены в том, как сделать грамотный выбор, рекомендую изучать. Также там есть расшифровка терминов игровых автоматов, что очень удобно для новичков. Когда я только начинала, было сложно понять все тонкости, но этот сайт действительно помог разобраться. Теперь я выбираю автоматы только там, потому что уверена в их надёжности. Последний раз я выиграла приличную сумму на автомате с бонусными раундами, которые оказались очень щедрыми. Вывод средств прошёл без проблем, что для меня очень важно
